src/ZendeskApi.Client/ZendeskApi.Client.csproj

```diff
@@ -99,30 +138,16 @@
     <Compile Include="Serialization\ZendeskJsonSerializer.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
+    <Folder Include="lib\" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\ZendeskApi.Contracts\ZendeskApi.Contracts.csproj">
-      <Project>{bea9e134-91ff-40d6-9b5d-bd57006dbbc6}</Project>
-      <Name>ZendeskApi.Contracts</Name>
-    </ProjectReference>
+    <None Include="App.config" />
+    <None Include="packages.config" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-  <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  -->
-  <Target Name="AfterBuild">
-    <Copy SourceFiles="$(OutputPath)$(AssemblyName).dll" DestinationFolder="$(OutputPath)..\..\lib\net45" ContinueOnError="true" Condition=" '$(Configuration)|$(Platform)|$(TargetFrameworkVersion)' == 'Release|AnyCPU|v4.5' " />
```

Comment:
These two after build copy steps, to copy the dlls from the release bin, to the \ZendeskApiClient\src\ZendeskApi.Client\lib\net45 folder are used by our CI Build to make a nuget package from. Could you put the two copy targets back.